src/procedures/ModifyMaxHolderPercentage.ts

```diff
@@ -8,9 +8,18 @@ import {
 } from '../types';
 import { PolymathError } from '../PolymathError';
 
+/**
+ * Procedure that modifies the maximum percentage that an individual token holder can hold
+ */
 export class ModifyMaxHolderPercentage extends Procedure<ModifyMaxHolderPercentageProcedureArgs> {
   public type = ProcedureType.ModifyMaxHolderPercentage;
 
+  /**
+   * - Set a new maximum percentage of token holders there can be
```

Comment:
This comment is wrong. It seems to be a copy of holder count. 
Also please refer always to security tokens instead to `token` when it applies